marklogic-data-hub-central/ui/e2e/cypress/e2e/mastering/e2eMasteringflow.cy.tsx

```diff
@@ -434,8 +433,12 @@ describe("Validate E2E Mastering Flow", () => {
       detailPage.clickBackButton();
     }
 
-    browsePage.getFacetItemCheckbox("collection", "sm-Patient-auditing").click();
+    browsePage.clearAllFacets();
+    browsePage.waitForSpinnerToDisappear();
     cy.waitForAsyncRequest();
```

Comment:
We don't need these commands since they are already inside the clearAllFacets method